app/src/main/java/org/systers/mentorship/utils/States.kt

```diff
@@ -0,0 +1,11 @@
+package org.systers.mentorship.utils
+
+/**
+ * A sealed class to represent error states
+ * in the EditProfileFragment. More error states
+ * can be added in the future as the application
+ * grows in complexity.
+ */
+sealed class EditProfileFragmentErrorStates {
+    object EmptyNameError: EditProfileFragmentErrorStates()
```

Comment:
Codacy is recommending to add KotlinDocs for this object [(link to Codacy)](https://app.codacy.com/app/m-murad/mentorship-android/pullRequest?prid=2718301)